Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -3574,8 +3580,8 @@ def fetch_incidents(client: MsClient, last_run, fetch_evidence):
             parsed = dateparser.parse(last_fetch_time, settings={'RETURN_AS_TIMEZONE_AWARE': True, 'TIMEZONE': 'UTC'})
             demisto.debug(f'Checking alert {alert["id"]} with parsed time {parsed}. last alert time is {alert_time}')
             if alert_time <= parsed:  # type: ignore
-                demisto.debug(f"{INTEGRATION_NAME} - alert {str(alert)} was created at {alert['alertCreationTime']}."
-                              f' Skipping.')
+                skipped_incidents += 1
+                demisto.debug(f"Microsoft - ATP - Skipping incident with id={alert[id]} because its creation time is smaller than the last fetch.")  # noqa: E501
```

Comment:
I think it is better just to print the time and id of the alert and not the whole object.